change/@fluentui-react-swatch-picker-preview-6cb1ebfe-d839-4335-adb5-483fc5a0ed27.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

this has no effect on runtime that we ship to npm thus  none

```suggestion
  "type": "none",
```